templates/zookeeper-env.sh.j2

```diff
@@ -0,0 +1,5 @@
+{% if zookeeper_env is defined %}
```

Comment:
Actually adding the check only in the template (only avoiding 'entering the loop') doesn't bring much (except saving <1ms).
IMO the proper fix is to add the check in the task, avoid to write an empty zookeeper-env.sh (if it's not used).  (With this change, one has to be aware that it will be more complicate to move from a file with settings to an emptied env file!)
```
- name: Configure zookeeper-env.sh
...
  when: zookeeper_env is defined and zookeeper_env|length  > 0
```
@ernestas-poskus ok for you (to only add that 'when' line ) ?